### ISSUE TITLE
Cannot create connections from SpikeSourceArray population with plastic synapses in NEST 2.6 backend

### ISSUE THREAD

**Issue by apdavison** [NeuralEnsemble/PyNN#368]
Example:
```
import pyNN.nest as sim
p1 = sim.Population(5, sim.SpikeSourceArray())
p2 = sim.Population(10, sim.IF_cond_exp())
syn = sim.TsodyksMarkramSynapse()
prj = sim.Projection(p1, p2, sim.AllToAllConnector(), syn)
```
gives
```
Jul 07 11:50:47 ConvergentConnect [Warning]: 
    Target with ID 10 does not support the connection. The connection will be ignored.
    Details: Creation of connection is not possible because:
    Possible cause: only static synapse types may be used to connect devices.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/nest/projections.py", line 55, in __init__
    connector.connect(self)
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/nest/connectors.py", line 147, in connect
    return super(AllToAllConnector, self).connect(projection)
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/connectors.py", line 265, in connect
    self._connect_with_map(projection, connection_map)
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/connectors.py", line 231, in _connect_with_map
    self._standard_connect(projection, connection_map.by_column, distance_map)
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/connectors.py", line 213, in _standard_connect
    projection._convergent_connect(source_mask, col, **connection_parameters)
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/nest/projections.py", line 163, in _convergent_connect
    self._identify_common_synapse_properties()
  File "/home/andrew/env/pyNN08/local/lib/python2.7/site-packages/pyNN/nest/projections.py", line 191, in _identify_common_synapse_properties
    local_parameters = nest.GetStatus(sample_connection)[0].keys()
IndexError: tuple index out of range
```

**Comment by apdavison** [comment]
This is new behaviour with NEST 2.6. It didn't occur with earlier versions.

**Comment by apdavison** [comment]
This is related to #352

**Comment by apdavison** [comment]
fixed in a3327f834544e2f74920cfbfed9ac7cb99a99f35
